test/verify/check-machines

```diff
@@ -688,6 +688,7 @@ class TestMachines(MachineCase, StorageHelpers, NetworkHelpers):
             if not self.use_existing_volume:
                 # Choose storage pool
                 if not self.pool_type or self.pool_type not in ['iscsi', 'iscsi-direct']:
+                    b.wait_present("#vm-{0}-disks-adddisk-new-select-pool:enabled".format(self.vm_name))
```

Comment:
Is 'enabled' attribute really on this selector? Can't see it on the dom. Hmm

Reply:
Yes it is. It's passed here https://github.com/cockpit-project/cockpit/blob/master/pkg/lib/cockpit-components-select.jsx#L81 and then passed into traditional html <select>. Also it fixed tests for me locally.